esiclient/v1/port_forwarding.py

```diff
@@ -34,6 +36,15 @@ class Protocol(str, Enum):
     UDP = "udp"
 
 
+@dataclass
+class FwdSpec:
```

Comment:
Not a big deal, but this class does seem pretty similar to `PortSpec`, and it looks like PortSpec is only used when deleting port forwardings. Is it possible to somehow combine the two? "Not really" is fine!